jenkins/scripts/bare_metal_lab/files/reset_network.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+sudo killall named
+sudo systemctl restart dnsmasq
+#sudo systemctl restart named
```

Comment:
Do we need this for future reference or can be removed?
Also, I am not sure if I am getting the idea why we need to reset the network before running m3-dev-env scripts, would you mind explaining more why it is needed?

Reply:
I've added a comment to the script and annotated each command.